mozilla/fx-private-relay
# MPP-2279 Add area code search for relay number suggestions.

This PR fixes MPP-2279 

# New feature description

This PR adds the functionality of searching for numbers based on area code. Updated UI to reflect new changes in Figma. 

# Screenshot (if applicable)

<img width="593" alt="image" src="https://user-images.githubusercontent.com/3924990/188061270-6e9081fe-4722-4861-8e53-8dafa3c8ac97.png">
                                                                           
# How to test

Get to the part in the onboarding process where you start choosing your Relay Number. Try using different area codes or city names in the top-most input. Submitting that input's information should update the numbers suggested. 

# Checklist
